HIFIMAGS/HIFIMAGS-IOC-01App/Db/hifimags.db

```diff
@@ -0,0 +1,240 @@
+### Generic Records
+
+record(bo, "$(P)SIM")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM)")
+}
+
+record(bo, "$(P)DISABLE") 
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+### Operation Mode
+
+record(mbbo, "$(P)OPMODE:SP") {
+  field(DESC, "Operation Mode")
+  field(DTYP, "Soft Channel")
+  
+  field(ZRST, "Idle")
+  field(ONST, "High Field")
+  field(TWST, "Low Field")
+  field(THST, "Z Switching")
+  
+  info(archive, "VAL")
+  info(INTEREST, "HIGH")
+}
+alias("$(P)OPMODE:SP", "$(P)OPMODE")
+alias("$(P)OPMODE:SP", "$(P)OPMODE:RBV")
```

Comment:
Did you mean  :SP:RBV ?

Reply:
No, the OPMODE:RBV is for compliance with the existing LabVIEW code